### PR TITLE
WSTEAM1-1085: Alter semantics

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves JIRA [WSTEAM1-1085](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1085)

Overall changes
======
Alters the semantics of labels and its relation to buttons and form fields.

Code changes
======
- _Adds id functionality to Label component._
- _Changes aria attributes of form component._

### CODE REVIEW

**Comment by amoore108** [comment]
Are there any links to GEL/a11y references for this change? Just trying to work out exactly what difference this will make semanatically.

File: ws-nextjs-app/pages/[service]/send/[id]/FormField/File/index.tsx [hunk 1]
```diff
@@ -184,7 +184,9 @@ export default ({
 
   return (
     <>
-      <Label id={labelId}>{label}</Label>
+      <Label id={id} elementId={labelId}>
```
Comment (Isabella-Mitchell) [inline ws-nextjs-app/pages/[service]/send/[id]/FormField/File/index.tsx]: I'm wondering if it's a bit confusing that the `id` we pass isn't actually the id. Would it be more appropriate to name this parameter something like `forId`? I appreciate that means we'll need to update other instances of <Label> (Open to thoughts on this!)

Comment (amoore108) [reply ws-nextjs-app/pages/[service]/send/[id]/FormField/File/index.tsx]: I agree, makes more sense calling it something like `forId`.

Comment (shayneahchoon) [reply ws-nextjs-app/pages/[service]/send/[id]/FormField/File/index.tsx]: I agree too

**Review by amoore108 — APPROVED** [review]

**Review by Isabella-Mitchell — APPROVED** [review]
Thanks!

**Pull request merged**